src/actions/helpers/formatSighting.js

```diff
@@ -0,0 +1,66 @@
+import moment from 'moment';
```

Comment:
Not formatting passed form values because that would change the values in the form. Creating a new object instead for POST.